distribution/docs/src/main/resources/_contents/_broker/messaging-broker-app-contents.adoc

```diff
@@ -0,0 +1,17 @@
+==== Message Broker App
+
```

Comment:
We probably don't need this section header as it comes immediately after the "Broker Reference" header.

We should have a brief intro defining what the application is though.